tests/reporter/kubernetes.go

```diff
@@ -1104,6 +1104,10 @@ func (r *KubernetesReporter) executeContainerCommands(virtCli kubecli.KubevirtCl
 				failedExecuteCmdFmt,
 				command, target, stdout, stderr, err,
 			)
+
+			if strings.Contains(err.Error(), "not found") {
```

Comment:
Is there a reason not to use `errors.IsNotFound`?

Reply:
It isn't a `IsNotFound` type, it is created by
`fmt.Errorf("unable to upgrade connection: %s", responseError)`
the `not found` is a string that appears in both cases of container and pod not found (see the full lines please in the ref issue)

Reply:
What worries me is that we may catch another error with `not found` that is not one of the two this PR wants to catch.
On the other side, if the messages we want to catch will be changed (for example to "Not Found") the condition will stop working.

To solve at least the first issue let's use the full two messages.

Reply:
[Changes](https://github.com/kubevirt/kubevirt/pull/7815#issuecomment-1142236264)